src/rpc/blockchain.cpp

```diff
@@ -1258,6 +1258,9 @@ static void BIP9SoftForkDescPushBack(UniValue& softforks, const std::string &nam
         statsUV.pushKV("possible", statsStruct.possible);
         bip9.pushKV("statistics", statsUV);
     }
+    if (consensusParams.vDeployments[id].min_activation_height != 0) {
+        bip9.pushKV("min_activation_height", consensusParams.vDeployments[id].min_activation_height);
```

Comment:
Added docs for `min_activation_height` field; the docs for `height` don't need updating?

Reply:
:+1:  -- may as well always return it rather than having client software infer a value from absence.

This can be handled in a follow up PR IMO (which cleans up the BIP9 naming too, since we've diverged)